spec/controllers/records_controller_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe RecordsController, type: :controller do
+  let(:user) { create(:user) }
+  let(:record) { create(:record, user: user) }
```

Comment:
We need `valid_attributes` and `record`?

Reply:
Yes, we do. `record` is used for the show, edit and update endpoints, `valid_attributes` is used on the create action for example.

Reply:
Saquei